src/scss/components/card.scss

```
@@ -116,6 +116,34 @@
       object-fit: cover;
     }
 
+    .media-date {
```

```suggestion
    .media--date {
```
To make this consistent with other classes we have that extend the base `.media` class.